files/tarball_ingester_userdata.tpl

```diff
@@ -73,7 +73,7 @@ acm-cert-retriever \
 
 echo "Setup minio..."
 chmod u+x /opt/tarball_ingestion/tarball_ingestion_minio.sh
-/opt/tarball_ingestion/tarball_ingestion_minio.sh "${s3_artefact_bucket}"
+/opt/tarball_ingestion/tarball_ingestion_minio.sh "${s3_artefact_bucket}" "${minio_s3_bucket_name}"
```

Comment:
FYI, there's a bug in tarball_ingestion_minio.sh on line 51; could you push a fix for that in this PR too please?